app/domain/authentication/authn_jwt/identity_providers/identity_from_decoded_token_provider.rb

```diff
@@ -29,35 +29,42 @@ def jwt_identity
 
         # Checks if variable that defined from which field in decoded token to get the id is configured
         def identity_available
```

Comment:
this should be `identity_available?` as it's a boolean value. please see that you do this throughout the PR.

Reply:
Done